src/Commands/Issue.php

```diff
@@ -95,9 +91,9 @@ private function doExecute(Manager $args): Generator {
 
                 file_put_contents("{$path}/{$token}", $payload);
                 chown("{$path}/{$token}", $userInfo["uid"]);
-                chmod("{$path}/{$token}", 0660);
+                chmod("{$path}/{$token}", 0664);
```

Comment:
In my environment apache is not the owner of the files. Since the file will be accessible over HTTP anyway I don't think making it work readable is a problem.